package.json

```diff
@@ -8,12 +8,16 @@
     "build": "vite build",
     "preview": "vite preview"
   },
-  "devDependencies": {
-    "vite": "^4.2.0"
-  },
   "dependencies": {
-    "jquery": "^3.6.4",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "ress": "^5.0.2",
     "toastify-js": "^1.12.0"
```

Comment:
(⁉) Are these dependencies really needed?

Reply:
react and react-dom are dependencies for the React framework, of which this rewrite was built upon.
ress is a css reset to enable more precise control of CSS styling, and toastify-js adds a simple toast component to add notifications to users.